post.js

```diff
@@ -1,10 +1,25 @@
 async function loadPost(url) {
   try {
-    const response = await fetch(url);
+    const headers = new Headers({
+      Accept: "application/activity+json, application/ld+json, text/html",
```

Comment:
Maybe add an `application/json` to the list just in case too? Also mind putting the string into a CONST at the top?